packages/ui5-application-inquirer/test/unit/prompts/prompt-helpers.test.ts

```diff
@@ -133,20 +133,32 @@ describe('prompt-helpers', () => {
         };
         filteredPrompts = hidePrompts(prompts, promptOpts);
         expect(filteredPrompts.length).toEqual(12);
-        expect(filteredPrompts).toEqual(expect.not.arrayContaining([{ name: promptNames.addDeployConfig }]));
+        expect(filteredPrompts).toEqual(
+            expect.not.arrayContaining([{ name: promptNames.addDeployConfig, when: expect.any(Function) }])
+        );
         expect(filteredPrompts).toEqual(expect.not.arrayContaining([{ name: promptNames.skipAnnotations }]));
         expect(filteredPrompts).toEqual(expect.not.arrayContaining([{ name: promptNames.ui5Version }]));
 
         // More testing of prompt options (hide fn)
         promptOpts = {
             [promptNames.addDeployConfig]: {
-                hide: (isCap) => {
-                    return isCap;
+                hide: (isCap: boolean) => {
+                    return !isCap;
                 }
             }
         };
+        // show `addDeployConfig` prompt when isCap is true
         filteredPrompts = hidePrompts(prompts, promptOpts, true);
-        expect(filteredPrompts.length).toEqual(12);
-        expect(filteredPrompts).toEqual(expect.not.arrayContaining([{ name: promptNames.addDeployConfig }]));
+        expect(filteredPrompts.length).toEqual(13);
+        expect(filteredPrompts).toEqual(
+            expect.arrayContaining([{ name: promptNames.addDeployConfig, when: expect.any(Function) }])
+        );
+
+        // hide `addDeployConfig` prompt when isCap is false
```

Comment:
Is this comment correct...the numbers suggest the comments are the wrong way around?